cdp_scrapers/instances/seattle.py

```diff
@@ -395,40 +406,55 @@ def get_content_uris(self, legistar_ev: dict) -> list[ContentURIs]:
-                "span", id=re.compile(r"ctl\S*_ContentPlaceHolder\S*_lblLocation$")
-            ).text.lower()
-        ):
             try:
-                if parse_qs(urlsplit(video_page_url).query)["videoid"]:
-                    # video link contains specific videoid
-                    return self.parse_content_uris(video_page_url, event_short_date)
-            except KeyError:
-                pass
-
-        # at this point video_page_url points to generic video list page like
-        # http://www.seattlechannel.org/BudgetCommittee?Mode2=Video
-
-        return [
+                # Some meetings will have text like "Session II" in "Meeting location".
+                # For those, don't bother verifying video page URL.
+                # They are multi-session and we need to call get_video_page_urls()
+                if (
+                    "session ii"
+                    not in soup.find(
+                        "span",
+                        id=re.compile(r"ctl\S*_ContentPlaceHolder\S*_lblLocation$"),
+                    ).text.lower()
+                ):
+                    try:
+                        if parse_qs(urlsplit(video_page_url).query)["videoid"]:
+                            # video link contains specific videoid
+                            return self.parse_content_uris(
+                                video_page_url, event_short_date
+                            )
+                    except KeyError:
+                        pass
+
+                # at this point video_page_url points to generic video list page like
+                # http://www.seattlechannel.org/BudgetCommittee?Mode2=Video
+
+                return [
+                    uris
+                    # 1 web page per session video for this multi-session event
+                    for page_url in self.get_video_page_urls(
+                        video_page_url, event_short_date
+                    )
+                    # video and caption urls on the session video web page
+                    for uris in self.parse_content_uris(page_url, event_short_date)
+                ]
+            except VideoIdMismatchError:
+                return []
+
+        event_date = datetime.fromisoformat(legistar_ev[LEGISTAR_SESSION_DATE])
+        # Try first using 2-digit year without century and then as 4-digit with century
+        # to account for old and new date strings used on Seattle Channel
+        uris = get_uris_for_date(event_date, event_date.strftime("%y"))
+        uris = (
             uris
-            # 1 web page per session video for this multi-session event
-            for page_url in self.get_video_page_urls(video_page_url, event_short_date)
-            # video and caption urls on the session video web page
-            for uris in self.parse_content_uris(page_url, event_short_date)
-        ]
+            if any(uris)
+            else get_uris_for_date(event_date, event_date.strftime("%Y"))
+        )
+        return uris
```

Comment:
All the changes to this file is to try first m/d/yy and m/d/yyyy when comparing event date to date string on some video web page.